reactioncommerce/reaction
# Fix for #3289 and #3294 Instagram login issues

This PR fixes the account page when logging in using Instagram. This fixes issues #3289 and #3294.

The account page now shows the Instagram profile picture and handles the situation where an email does not exist due to Instagram no longer sending the user email over anymore when registering a user.

To test:

1. Login using Instagram service
2. Navigate to account page http://localhost:3000/account/profile
3. Observe that Instagram photo is now on the Account page and page does shows the correct fields.
